### ISSUE TITLE
Curation - don't display JIGs that have not yet been published

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#2708]
Currently, all JIGs are being displayed, and if not published, then they have no name or thumbnails. This is going to lead to problems when a curator republishes, without knowing that it is a draft that someone is working on.
Please only display PUBLISHED JIGs in the curation table.

![image](https://user-images.githubusercontent.com/16290361/166889241-dafd5dd1-3f75-48e8-a4ae-bd59b84b9790.png)

**Comment by corinnewo** [comment]
@johnnynotsolucky Were you able to look into hiding the draft JIGs in curation?

**Comment by johnnynotsolucky** [comment]
@corinnewo I didn't get a chance yet. Ricky said he'll be able to take care of it.

@RickyLB, the browse endpoint gets `DraftOrLive::Live`, and returns the empty live jig_data and the jigs have published_at null. So i'm guessing the query just looking for live records whether they've been published or not.

Example request:

```
GET https://api.sandbox.jigzi.org/v1/jig/browse?page=0&jigFocus=modules&draftOrLive=live
```

**Comment by johnnynotsolucky** [comment]
Closed in #2745
